src/vs/workbench/parts/terminal/electron-browser/terminalPanel.ts

```diff
@@ -27,6 +27,18 @@ import { StandardMouseEvent } from 'vs/base/browser/mouseEvent';
 import { TPromise } from 'vs/base/common/winjs.base';
 import URI from 'vs/base/common/uri';
 
+// List of regular expressions, for matching shells, functions to format a path, and booleans indicating whether the shell uses Windows-style paths
+const WIN_PATH_FORMATTERS: [RegExp, (path: string) => string, boolean][] = [
```

Comment:
Let's use an interface instead of an array so it's nicer to use:

```ts
interface IPathFormatter {
  regex: RegExp,
  format: (path: string) =>string,
  usesWindowsStylePaths: boolean
}

const WIN_PATH_FORMATTERS: IPathFormatter[] = [...]
```